src/ts/uifactory.ts

```diff
@@ -238,14 +237,18 @@ export namespace UIFactory {
         controlBar,
         new TitleBar({
           components: [
-            new MetadataLabel({ content: MetadataLabelContent.Title }),
-            new CastToggleButton(),
-            new VRToggleButton(),
-            new PictureInPictureToggleButton(),
-            new AirPlayToggleButton(),
-            new VolumeToggleButton(),
-            new SettingsToggleButton({ settingsPanel: settingsPanel }),
-            new FullscreenToggleButton(),
+            new Container({
```

Comment:
Again `Container` within `Container` - looks like a hack. There should be a cleaner solution. What is the actual issue why this is needed?